cmd/conduit/main.go

```diff
@@ -7,18 +7,15 @@ import (
 	"os"
 	"path/filepath"
 
+	"github.com/algorand/indexer/util/metrics"
 	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 
 	"github.com/algorand/indexer/conduit"
-	"github.com/algorand/indexer/loggers"
-)
-
-import (
-	// We need to import these so that the package wide init() function gets called
```

Comment:
Nit: I think this is a useful comment in relation to the `_` imports--explains why they're there.

Reply:
my code editor keeps merging these import blocks into a single block. it seems that gofmt doesn't support using multiple import blocks.

Reply:
updated.